modules/dynamic-plugins/proc-package-third-party-dynamic-plugin.adoc

```diff
@@ -0,0 +1,158 @@
+[id="proc-package-third-party-dynamic-plugin_{context}"]
+= Packaging and publishing third-party plugins as dynamic plugins
+
+To package a third-party plugin as a dynamic plugin, you need to access to its source code and must export a plugin using the `@janus-idp/cli`. For more information, see xref:proc-export-third-party-plugins-rhdh_{context}[]. 
+
+After exporting a third-party plugin, you can package the derived package into one of the following supported formats:
+
+* Open Container Initiative (OCI) image (recommended)
+* TGZ file
+* JavaScript package
+
+[IMPORTANT]
+====
+Exported dynamic plugin packages must only be published to private NPM registries.
+====
+
+[#convert-to-oci-image]
+== Converting a dynamic plugin package to an OCI image
+
+.Prerequisites
+* You have installed `podman` or `docker`.
+* You have exported a third-party dynamic plugin package. For more information, see xref:proc-export-third-party-plugins-rhdh_{context}[].
+
+.Procedure
+. Navigate to the exported plugin's root directory (not the `dist-dynamic` directory).
+. Run the following command to package the plugin into an OCI image:
++
+--
+.Example command to package an exported third-party plugin
+[source,bash]
+----
+npx @janus-idp/cli@latest package package-dynamic-plugins --tag quay.io/example/image:v0.0.1
+----
+
+In the previous command, the `--tag` argument specifies the image name and tag.
+--
+. Run one of the following commands to push the image to a registry:
++
+--
+.Example command to push an image to a registry using podman
+[source,bash]
+----
+podman push quay.io/example/image:v0.0.1
+----
+
+.Example command to push an image to a registry using docker
+[source,bash]
+----
+docker push quay.io/example/image:v0.0.1
+----
+
+The output of the previous commands provides the plugin's path for use in the `dynamic-plugin-config.yaml` file.
+--
+
+[#convert-to-tgz]
+== Converting a dynamic plugin package to a TGZ file
```

Comment:
```suggestion
== Packaging a dynamic plugin packages to a TGZ file
```

Reply:
Changed it to "== Creating a TGZ file with dynamic packages"